pyqtgraph/dockarea/Dock.py

```diff
@@ -262,7 +262,27 @@ class DockLabel(VerticalLabel):
     sigClicked = QtCore.Signal(object, object)
     sigCloseClicked = QtCore.Signal()
 
-    def __init__(self, text, closable=False, fontSize="12px"):
+    def __init__(self, text, closable=False, fontSize=12, tabColor='#4a5c96'):
+        self.r = '5px'  # Tab radius
+        if isinstance(tabColor, str) and tabColor.startswith('#') and len(tabColor) ==7:
```

Comment:
I suggest using `pg.mkColor(tabColor)` here rather than implementing your own color parsing.

Reply:
yeah, definitely use `mkColor` as it handles all sorts of formats.